### ISSUE TITLE
Weather.com forecast is not working

### ISSUE THREAD

**Issue by Santanachia** [pkissling/clock-weather-card#238]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser
- [x] I read the [FAQ](https://github.com/pkissling/clock-weather-card#faq) and the issue I'm having was not mentioned there.

**Release with the issue:**
2.0.0

**Last working release (if known):**

**Browser and Operating System:**
Edge v117
Windows 11

**Description of problem:**
After updating HA to version 2023.10.1, the forecast stopped displaying. I think it may have to do with changes in Weather.com integration (Or in the HA itself?), rather than the clock-weather-card

**Javascript errors shown in the web inspector (if applicable):**
```
clock-weather-card - Error while rendering clock-weather-card component: Error: Weather entity does not have attribute "forecast".
```

**Additional information:**
Although there is no `forecast` attribute, the forecast is available
![image](https://github.com/pkissling/clock-weather-card/assets/4628507/0b1434d3-0316-4957-8690-2f1061761bc2)
![image](https://github.com/pkissling/clock-weather-card/assets/4628507/9edc64b1-c14d-4009-a38b-2853c6ceb3d3)

**Comment by Santanachia** [comment]
p.s.
```yaml
service: weather.get_forecast
data:
  type: daily
target:
  entity_id: weather.zaciszna
  ```
  ```yaml
  forecast:
  - cloud_coverage: 23
    condition: partlycloudy
    precipitation: 0
    precipitation_probability: 1
    temperature: 13
    templow: 6
    datetime: "2023-10-10T05:00:00Z"
    wind_bearing: pd.
    wind_speed: 12
  - cloud_coverage: 82
    condition: cloudy
    precipitation: 0
    precipitation_probability: 19
    temperature: 22
    templow: 16
    datetime: "2023-10-11T05:00:00Z"
    wind_bearing: pd.pd.-zach.
    wind_speed: 24
...
```

**Comment by pkissling** [comment]
Either you did not update the plugin to the latest version or your browser still uses the cached value, but the issue is fixed since version `1.1.0`. the error message you posted `Weather entity does not have attribute "forecast".` does not even exist anymore since `1.1.0`
